src/sass/components/_tour.scss

```diff
@@ -37,14 +37,73 @@
 
 .popover-wrapper {
   position: absolute;
-  top: -10px;
+  left: -($spacing-xx-small);
+  top: -($spacing-small);
+
+  .epic-progress & {
+    top: -($spacing-medium);
+  }
 
   .logo-popover & {
-    top: unset;
+    top: $spacing-small;
+    left: $spacing-large;
+  }
+
+  .metecho-breadcrumb & {
+    left: $spacing-large;
+    top: -($var-spacing-x-small);
+  }
+
+  .project-placeholder & {
+    top: $spacing-medium;
+  }
+
+  .project-list & {
+    left: $spacing-none;
+    top: $spacing-xxx-small;
   }
 
   .page-title & {
-    top: 25px;
+    left: $spacing-large;
+    top: $var-spacing-medium;
+  }
+
+  .user-info & {
+    left: -($spacing-xx-small);
+    top: -($spacing-small);
+  }
+
+  .page-options & {
+    left: -($spacing-xx-small);
+    top: -($spacing-medium);
+  }
+
+  .vertical-right & {
+    right: -($spacing-x-small);
+    top: -($spacing-medium);
+  }
```

Comment:
@dvdherron This is another button that has a slightly different positioning from some of the other buttons. If there is a reason to keep this one different than the others, maybe we can document why there are some that benefit from the position overrides. If this isn't needed, we will want to remove it along with the place it is used in the list.tsx file since I think that is the only place it is used.

Reply:
this was something I missed from the transition from right to left placement. removed.